### PR TITLE
[get-user-data] Add all outputs to context

### PR DESCRIPTION
**Pull request by mmhw**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Related: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-12149)

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bziser — APPROVED** [review]

File: Packs/CommonScripts/Scripts/GetUserData/GetUserData.py [hunk 1]
```diff
@@ -435,19 +469,19 @@ def ad_get_user(command: Command) -> tuple[list[CommandResults], dict[str, Any],
     )
     output_key = get_output_key("ActiveDirectory.Users", entry_context[0])
     outputs = get_outputs(output_key, entry_context[0])
-
-    account_disable = outputs.get("userAccountControlFields", {}).get("ACCOUNTDISABLE")
+    account_disable = outputs.get("userAccountControlFields", {}).pop("ACCOUNTDISABLE", None)
     is_enabled = (not account_disable) if isinstance(account_disable, bool) else None
+    manager_dn = (outputs.pop("manager") or [""])[0]
```
Comment (mmhw) [inline Packs/CommonScripts/Scripts/GetUserData/GetUserData.py:474]:
```suggestion
    manager_dn = (outputs.pop("manager", None) or [""])[0]
```

**Comment by kobymeir** [comment]
### This PR was automatically updated by a [GitHub Action](https://github.com/demisto/content/actions/runs/12115398515)
- **CommonScripts** pack version was bumped to **1.15.97**.

To stop automatic version bumps, add the `ignore-auto-bump-version` label to the github PR.

**Comment by kobymeir** [comment]
### This PR was automatically updated by a [GitHub Action](https://github.com/demisto/content/actions/runs/12119706342)
- **CommonScripts** pack version was bumped to **1.15.98**.

To stop automatic version bumps, add the `ignore-auto-bump-version` label to the github PR.

**Comment by kobymeir** [comment]
### This PR was automatically updated by a [GitHub Action](https://github.com/demisto/content/actions/runs/12122003002)
- **CommonScripts** pack version was bumped to **1.15.99**.

To stop automatic version bumps, add the `ignore-auto-bump-version` label to the github PR.

**Pull request merged**